build-tools/tasks/test-utils.js

```diff
@@ -57,12 +105,22 @@ function generateIndexFileContent(testUtilType, testUtilMetaData) {
     ...testUtilMetaData.map(({ componentName }) => {
       const wrapperName = toWrapper(componentName);
       // language=TypeScript
-      return `ElementWrapper.prototype.find${componentName} = function(selector) {
-          const rootSelector = \`.$\{${wrapperName}.rootSelector}\`;
-          // casting to 'any' is needed to avoid this issue with generics
-          // https://github.com/microsoft/TypeScript/issues/29132
```

Comment:
Is this not needed anymore? Why?

Reply:
This issue is related to the `protected` properties. All of the generated functions are `public` so not sure why this has been an issue to begin with.

Reply:
If we don't know why this is here, it does not sound safe to me to remove it - we can't say with certainty whether the original reason still exists or not, and we don't know how to test for it. I would recommend to keep this casting.

Reply:
To err on the safer side, I'll keep this for `findX` function. We can fix this in another scope.